examples/docs/en-US/datetime-picker.md

```diff
@@ -117,6 +118,15 @@ DateTimePicker is derived from DatePicker and TimePicker. For a more detailed ex
       :picker-options="pickerOptions1">
     </el-date-picker>
   </div>
+  <div class="block">
+    <span class="demonstration">With default time</span>
+    <el-date-picker
+      v-model="value3"
+      type="datetime"
+      placeholder="Select date and time"
+      :default-time="'12:00:00'">
```

Comment:
How about `default-time="12:00:00"`? It is cleaner

Reply:
Made above change and resubmitted.